src/main/java/net/syscon/elite/repository/jpa/model/ReferenceCode.java

```diff
@@ -16,16 +18,26 @@
 @Entity(name = "REFERENCE_CODES")
 @DiscriminatorColumn(name = "domain")
 @Inheritance
+@IdClass(ReferenceCode.Pk.class)
 public abstract class ReferenceCode implements Serializable {
+
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class Pk implements Serializable {
+        private String domain;
+
+        private String code;
+    }
+
     @Id
+    @Column(insertable = false, updatable = false)
     private String domain;
+
     @Id
     private String code;
 
     private String description;
 
-    static final String CASE_STS = "CASE_STS";
-    static final String LEG_CASE_TYP = "LEG_CASE_TYP";
     static final String MLTY_BRANCH = "MLTY_BRANCH";
```

Comment:
should these still be here

Reply:
I really wanted to keep the changes specific to court stuff.  Was planning on do this in a separate PR.